src/docs/installing/post-install.md

```diff
@@ -59,6 +59,98 @@ If you experience issues in applications such as Parsec, or encounter disruptive
 3. Type `grub-mkconfig -o /boot/grub/grub.cfg` or `update-grub` into a terminal and press Enter. Use sudo, su, or doas if necessary.
 4. Reboot
 
+### Fixing Stylus on Rotation Issue
+
+Some Chromebooks come with USI styluses. They work fine in the newest versions of KDE Plasma, but they have a rotation issue in GNOME. The issue is that the stylus does not rotate with the screen, so it is only usable in one orientation. To fix this, we have to add an libinput override.
+
+1. Get the device ID of your stylus by running this script https://github.com/linuxwacom/libwacom/blob/master/tools/show-stylus.py.
```

Comment:
Name/model of the digitizer could also be read from cbmem, it's always connected over i2c on these machines

Reply:
Would that be better, since ,as far as I know, this would require the cbmem tool?

Reply:
Yes it would, but mentioning it would suffice for now

Reply:
Like this?

This information can either be read from cbmem or libwacom.
To get the ID from libwacom run this script https://github.com/linuxwacom/libwacom/blob/master/tools/show-stylus.py.

Reply:
LGTM